app/assets/stylesheets/common/base/compose.scss

```diff
@@ -307,6 +307,11 @@
         margin-right: 5px;
       }
     }
+    #draft-status .d-icon-user-edit {
+      color: $danger;
+      font-size: 20px;
+      vertical-align: -5.5px;
```

Comment:
Not a big fan of this kind of magic css value, can’t we do anything better?

Reply:
Not as far as I know with my css knowledge